article/app/views/liveblog/liveBlogBody.scala.html

```diff
@@ -108,6 +107,36 @@ <h1 itemprop="headline" class="content__headline js-score">@Html(article.trail.h
                         <div class="after-article js-after-article"></div>
 
                         <div class="js-bottom-marker"></div>
+                    } else {
+                        @fragments.dropdown("Live feed", Some("live-feed"), true) {
+                            <div class="js-article__container" data-component="body">
+                                <div class="toast__space-reserver">
+                                    <div id="toast__tofix" class="toast__container">
+                                        <button class="toast__button toast__button--closed button button--large">
+                                            <span class="toast__text"></span>
+                                            @fragments.inlineSvg("refresh", "icon", List(""))
+                                        </button>
+                                    </div>
+                                </div>
+                                @if(model.currentPage.currentPage.isArchivePage) {
+                                    @fragments.liveBlogNavigation(article.content.metadata.id, model.currentPage)
+                                }
+                                <div class="js-liveblog-body u-cf from-content-api js-blog-blocks blocks @if(article.fields.isLive) {live-blog}" data-most-recent-block="block-@{
+                                    _root_.liveblog.LatestBlock(article.blocks)
+                                }" data-test-id="live-blog-blocks"
+                                itemprop="@if(article.tags.isReview) {reviewBody} else {articleBody}">
+                                @views.html.liveblog.liveBlogBlocks(model.currentPage.currentPage.blocks, article, timezone)
+                                </div>
+                                @* Remove LiveFilter and the fragment once Toast comes out of AB test status *@
+                                @fragments.liveBlogNavigation(article.content.metadata.id, model.currentPage)
+                            </div>
+
+                            @fragments.submeta(article)
+
+                            <div class="after-article js-after-article"></div>
+
+                            <div class="js-bottom-marker"></div>
+                        }
```

Comment:
Good shout. I could probably extract everything inside the dropdown into a fragment